docs/upgrade.html

```diff
@@ -62,7 +62,6 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             the software and metadata versions must be at least 3.3.x (the first version when KRaft mode was deemed production ready). For clusters in KRaft mode
             with versions older than 3.3.x, we recommend upgrading to 3.9.x before upgrading to 4.0.x. Clusters in ZooKeeper mode
             have to be migrated to KRaft mode before they can be upgraded to 4.0.x as described below.
-            <!--#include virtual="zk2kraft.html" -->
```

Comment:
We probably need to change the sentence above slightly. Maybe we should say something like:

> Clusters in ZooKeeper mode have to be <a href="..">migrated to KRaft mode</a> before they can be upgraded to 4.0.x as described below.

And the link should be to the migration to kraft page in 3.9.

Reply:
The link would be: https://kafka.apache.org/39/documentation/#kraft_zk_migration

Reply:
IIUC, we hardcode the URL to redirect to version 3.9, the `ZooKeeper to KRaft Migration` section, right?

Reply:
We seem to have lost this change?

Reply:
Sorry for the inconvenience.
I have added it back.